src/index.ts

```diff
@@ -1,27 +1,16 @@
 import { NativeModules } from 'react-native'
-import fallback from 'base64-js'
 
 const Base64Module = NativeModules.QuickBase64
 
-if (Base64Module && typeof Base64Module.install === 'function') {
+if (Base64Module && typeof global.base64FromArrayBuffer !== 'function') {
```

Comment:
No need to check for .install, it will be available. The guard is only for fast refreshes or hot reloads, so it doesn't call install again if the global still exists.